requirements.txt

```diff
@@ -3,6 +3,6 @@ mkdocs-exclude==1.0.2
 mkdocs-material==9.5.22
 mkdocs-glightbox==0.4.0
 mkdocs-material-extensions==1.3.1
-mkdocs-build-plantuml-plugin==1.11.0
+mkdocs-build-plantuml-plugin==1.9.0
```

Comment:
Why is this downgraded?

Reply:
Version `1.10.0` has been yanked and version `1.11.0` is python 3.12 only. This container only runs on python 3.11.